lib/Admin.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+/**
+ * The dashboard-specific functionality of the plugin.
+ *
+ * @link       http://log.pt/
+ * @since      1.0.0
+ *
+ * @package    RestKeyAuth
+ * @subpackage RestKeyAuth/lib
+ */
+
+namespace logoscon\RestKeyAuth;
+
+/**
+ * The dashboard-specific functionality of the plugin.
+ *
+ * Defines the plugin name, version, and two examples hooks for how to
+ * enqueue the dashboard-specific stylesheet and JavaScript.
+ *
+ * @package    RestKeyAuth
+ * @subpackage RestKeyAuth/lib
+ * @author     log.OSCON, Lda. <engenharia@log.pt>
+ */
+class Admin {
+
+	/**
+	 * The plugin's instance.
+	 *
+	 * @since  1.0.0
+	 * @access private
+	 * @var    Plugin $plugin This plugin's instance.
+	 */
+	private $plugin;
+
+	/**
+	 * Initialize the class and set its properties.
+	 *
+	 * @since 1.0.0
+	 *
+	 * @param Plugin $plugin This plugin's instance.
+	 */
+	public function __construct( Plugin $plugin ) {
+		$this->plugin = $plugin;
+	}
+
+	/**
+	 * The primary handler for user authentication.
+	 *
+	 * @since     0.1.0
+	 * @param     mixed    $user    The current user (or bool) passing through the filter.
```

Comment:
This boolean should always be false, otherwise the return will be `true` right on the first check. Is there a problem with this or is it "impossible" to get `true` coming in?